GSG-K3/360HMR
# Navbar

### What does this PR do?
- Front-end Navbar on the home page

### Does this Fix an Issue?
Fixes #54 

### How to test this PR
- npm run dev don't forget to npm i on both sides 

- check the navbar on the home page "/" and also on the dashboard page '/dashboard' just to check the difference between the home page and the rest of the pages 

### Screenshots of changes

![navbar](https://user-images.githubusercontent.com/39573452/87410462-04580180-c5cf-11ea-955c-64dccc5822b0.png)
